Frameworks/include/Platform/EbrPlatform.h

```diff
@@ -38,17 +38,14 @@ SB_EXPORT int EbrFflush(int fd);
 SB_EXPORT int EbrOpen(const char* file, int mode, int share);
 SB_EXPORT int EbrOpenWithPermission(const char* file, int mode, int share, int pmode);
 SB_EXPORT int EbrClose(int fd);
-SB_EXPORT int EbrFd2Host(int fd);
 SB_EXPORT int EbrFstat(int fd, struct stat* ret);
 SB_EXPORT int EbrFstat64i32(int fd, struct _stat64i32* ret);
 SB_EXPORT int EbrRead(int fd, void* dest, size_t count);
 SB_EXPORT int EbrWrite(int fd, const void* src, size_t count);
 SB_EXPORT int EbrLseek(int fd, __int64 pos, int whence);
```

Comment:
>int [](start = 10, length = 3)

@DHowett-MSFT @msft-Jeyaram  still working on this bit. Got very much sidetracked from test failures due to ridiculously wrong NSCondition use.